client/blocks/eligibility-warnings/index.jsx

```diff
@@ -87,7 +87,7 @@ const EligibilityWarnings = props => {
 
 	const classes = classNames( {
 		'eligibility-warnings__message': true,
```

Comment:
Aside: Keys in `classNames` whose values are always `true` can alternatively be passed as separate string arguments to the function, i.e.:

```js
classNames( 'eligibility-warnings__message', {
	'is-placeholder': isPlaceholder
} );
```